Basic-Video-Chat-XCFramework/README.md

```diff
@@ -1,6 +1,8 @@
 Basic Video Chat sample app
 ===========================
 
+**NOTE:** *OTXCFramework only exists for version 2.20.0 +*
```

Comment:
How about "for versions 2.20.0 and above"?

Reply:
updated!